### ISSUE TITLE
WCAG2Mobile: Draft feedback - 2.1.2 No Keyboard Trap

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#659]
(Mary Jo) No comment for now...We will have to follow the MATF continuing discussions on Issue 30.

- https://w3c.github.io/matf/#success-criterion-2-1-2-no-keyboard-trap
- https://github.com/w3c/matf/issues/30

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
